hardware/raspberrypi/bootmodes/net.md

```diff
@@ -1,8 +1,7 @@
 # Network booting
 
-**Network booting is available on Raspberry Pi 3B, 3B+, and 2B v1.2 only.**
-
-This section describes how network booting works. We also have a [tutorial about setting up a working bootable system](net_tutorial.md). Network booting works only for the wired adapter built into the above models of Raspberry Pi. Booting over wireless LAN is not supported, nor is booting from any other wired network device.
+This section describes how network booting works on the Raspberry Pi 3B, 3B+ and 2B v1.2. On the Raspberry Pi 4, network booting is implemented in the second stage bootloader in the EEPROM. Please see the [Pi 4 Bootloader Configuration](../bcm2711_bootloader_config.md) section for more information.
```

Comment:
section -> page

Reply:
I assume that it's the _second_ mention of 'section' here (near the end of the sentence) that @andrum99 is referring to?

Reply:
Yes. So:

see the Pi 4 Bootloader Configuration section for more information -> see the Pi 4 Bootloader Configuration page for more information